### PR TITLE
Miscellaneous article formatting clean-ups

### PR DESCRIPTION
**Pull request by TPGPL**
- Removed MediaWiki's bold syntax
- Cleaned up the rest of wrongly parsed articles
- Fixed header levels

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 1]
```diff
@@ -85,10 +85,10 @@ Voici un exemple comportant des jackhammers : [MENDES Black Another](https://www
 Ces informations sont importantes et devraient être suivis par la plupart des mappers. Cependant, ce ne sont pas des règles, ce qui veut dire qu'un mapper peut choisir de les ignorer sous certaines circonstances. Si vous êtes sur le point d'ignorer un conseil, posez vous des questions : "Est-ce que ce que je m'apprête à apporter quelque chose de positif à ma map ?" Soyez certain de ce que vous faites.
 
 -   **Le HP Drain de la map devrait être placé à 7 si vous souhaitez en avoir un équivalent à *IIDX*.**
--   '''L'Overall Difficulty devrait être réglé sur :\*
-    -   - 9 ou 10 pour une map composée presque exclusivement de Hits.
-    -   - 7 ou 8 pour une map composée de Holds et de Hits de façon équilibrée.
-    -   - 5 ou 6 pour une map composée de nombreux Holds, telles que les maps Insane des jeux O2.
+-   **L'Overall Difficulty devrait être réglé sur :\***
```
Comment (cl8n) [inline wiki/Ranking_Criteria/osu!mania/fr.md:88]: asterisk at the end looks like a mistake

File: wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md [hunk 2]
```diff
@@ -2,32 +2,6 @@
 outdated: true
 ---
 
-# Traitement des erreurs
```
Comment (cl8n) [inline wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md:5]: why remove im not following

Comment (TPGPL) [reply wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md:5]: changed according to this 

http://web.archive.org/web/20160819223414/http://osu.ppy.sh/wiki/Handling_Cheating/Foul_Play

looks like somehow two seperate articles were put inside a single file so i deleted the one that surely shouldn't be in *this* folder

Comment (cl8n) [reply wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md:5]: if its not copied from somewhere else on the wiki maybe throw it in meta/unused then

Comment (TPGPL) [reply wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/fr.md:5]: done

**Pull request merged**